admin/app/components/solidus_admin/ui/modal/component.html.erb

```diff
@@ -23,7 +23,7 @@
         ) %>
       </header>
 
-      <div class="p-4 overflow-auto max-h-96">
+      <div class="p-4 overflow-auto max-h-97">
```

Comment:
This is not a supported TW value, doing this will effectively remove a max-height from this element, which could be fine if that's what we want.